### PR TITLE
WIP: Change host URL by checking the browser

### PR DESCRIPTION
**Pull request by anishsarangi**
In this PR , I have changed the host url by checking the browser. This is still under development.

### CODE REVIEW

**Comment by cgorringe** [comment]
I wonder if `InstallTrigger` would work in the future? Would using the user-agent string be better, such as:
`var isFirefox = (navigator.userAgent.indexOf("Firefox") !== -1)`

**Comment by anishsarangi** [comment]
Thank you @cgorringe . I agree, it would be better if I will user the `userAgent` string  instead of `InstallTrigger`

**Comment by cgorringe** [comment]
I haven't tested this yet @anishsarangi, but if it's ready, take it out of Draft if you want me to test and merge it.

**Comment by anishsarangi** [comment]
No, I am still working on it. Will let it know when it is ready for review ! Thank you !

**Comment by anishsarangi** [comment]
@cgorringe  WE can merge this now so that we can test all the endpoints !

**Comment by cgorringe** [comment]
OK, merging so we can test it on master!

File: scripts/background.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var VERSION = manifest.version
 var globalStatusCode = ''
 let toolbarIconState = {}
 let tabIdPromise
-var WB_API_URL = 'https://archive.org/wayback/available'
+var WB_API_URL = hostURL+'wayback/available'
```
Comment (cgorringe) [inline scripts/background.js:17]: Already merged, but found one possible bug here, as this may be defined before hostURL is?

Comment (anishsarangi) [reply scripts/background.js:17]: @cgorringe Actually utils.js is loaded before background.js . You can check in manifest.json file lineNo-37

**Comment by anishsarangi** [comment]
@cgorringe As there are a lot of APIs we are using in the extension, I am thinking of defining all the URL in the utils.js itself and use that const variable in other places directly. That will increase the code readability. What do you think of this ?

**Pull request merged**